script-loader.js

```diff
@@ -0,0 +1,43 @@
+import {getQueryParams} from './helpers.js';
+
+/**
+ * Loads script
+ *
+ * @param url {String} the url of the JS source
+ * @param type {String} type of script
+ * @param async {boolean}
+ * @param callback {function}
+ */
+function loadScript(url, type, async, callback){
+
+    var script = document.createElement("script");
+    script.type = type;
```

Comment:
I don't think module is needed.

Reply:
Static import statement cannot be used if the script type is not module.